### PR TITLE
[ADD] IoT: add device opc-ua protocol

### PR DESCRIPTION
**Pull request by tiku-odoo**
Creating new doc per task: 
https://www.odoo.com/web#id=3292030&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

New OPC-UA protocol is now available for use on the IoT box. New doc will explain what OPC-UA is and announce it's availability with IoT. 

Still exploring API connections and will add in as more information is available.

### CODE REVIEW

**Comment by lse-odoo** [comment]
@tiku-odoo to be fair, this is the first time that I am hearing about this and I have some doubts on the way it works.

At the end of your documentation you mention the usage of XML-RPC to connect machine to the IoT box (if I understood well). However I don't really see how it can be possible in practice as XML-RPC connections with the IoT box require a login, password & database and the Odoo service running on the IoT box does NOT use any database (so no login & password neither). Am I missing something ? Like, the connection should be done to the .odoo.com database and not the IoT box ?

**Comment by tiku-odoo** [comment]
@StraubCreative This new doc is ready for your review when you have a moment. It is a new feature, never added to our docs in V15. We have limited information on the feature but I think it is good for clients to know.

Setting as priority addition. 

Thanks for your review. 

Tim

**Comment by tiku-odoo** [comment]
> @tiku-odoo can you make sure it is also reviewed by the R&D IoT team as I have no idea what this thing is about ?

Sure, I have reached out to @dama-odoo the IoT PO to see if they can provide some examples.

**Comment by tiku-odoo** [comment]
@dama-odoo 

When you have a moment, can you review these changes? 

Thanks.

**Comment by dama-odoo** [comment]
Hi @tiku-odoo, apology for the late answer, feel free to reach out in Discord if I don't answer right away.
To me, it's sounds good but I need to redirect to @qle-odoo to confirm the content is alright, as I'm not an OPC-UA expert.

**Comment by tiku-odoo** [comment]
@dama-odoo QLE hasn't responded. Are you are to connect with them to see if they can review this document? 

Thanks. Tim

**Comment by qle-odoo** [comment]
@tiku-odoo sorry i check it tomorrow

**Comment by tiku-odoo** [comment]
Bumping this up. @dama-odoo @qle-odoo 

We are getting questions on this feature, and I think it would be good to get documented. Do you have any further details? Thanks :+1:

**Comment by StraubCreative** [comment]
@tiku-odoo since this information doesn't cover a specific feature or flow, and is more of a general FYI, I think this might make a better fit on the IoT parent page (`applications/productivity/iot.rst`) with a custom anchor for easy reference, instead of being it's own separate doc.

Are you open to this?

**Comment by tiku-odoo** [comment]
> @tiku-odoo since this information doesn't cover a specific feature or flow, and is more of a general FYI, I think this might make a better fit on the IoT parent page (`applications/productivity/iot.rst`) with a custom anchor for easy reference, instead of being it's own separate doc.
> 
> Are you open to this?

Yes, I'm open to this. I hoped for more information on how to integrate into the IoT. I can make this an informational admonition block on the main IoT page until we gain more guidance from our IoT PO.  Thanks for bumping this doc to the surface... :+1:

**Comment by StraubCreative** [comment]
> Yes, I'm open to this. I hoped for more information on how to integrate into the IoT. I can make this an informational admonition block on the main IoT page until we gain more guidance from our IoT PO. Thanks for bumping this doc to the surface... 👍

Cool let's do that for now wherever you see fit and I should be able to approve that one quickly since it's just a block of content 😉 

Anything further than that block can be addressed in a new Task / future PR.

**Comment by tiku-odoo** [comment]
@StraubCreative 

This PR is ready for your review and merge when you have a moment. 
Thanks.
Tim

**Comment by qle-odoo** [comment]
It's good for me.
OPC-UA is a very specific functionality.
And only on demand.
cc @lse-odoo

File: content/applications/productivity/iot.rst [hunk 1]
```diff
@@ -12,3 +12,14 @@ Internet of Things (IoT)
 
    iot/config
    iot/devices
+
+.. seealso::
```
Comment (StraubCreative) [inline content/applications/productivity/iot.rst:16]: Think a **note** block is better here [to convey additional information like this](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#note).

**seealso** is mostly used to link references and other documentation, so doesn't *quite* fit the use case here I don't think.

**Review by StraubCreative — APPROVED** [review]
Hi @tiku-odoo 
Quick change below which I'll push up and merge, thanks!

**Comment by StraubCreative** [comment]
@robodoo r+

**Pull request closed without merging**